presses-universitaires-de-paris-nanterre.csl

```diff
@@ -0,0 +1,592 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" default-locale="fr-FR" version="1.0" page-range-format="expanded" demote-non-dropping-particle="display-and-sort">
+  <info>
+    <title>Presses universitaires de Paris Nanterre (note, French)</title>
+    <title-short>PU Paris Nanterre</title-short>
+    <id>http://www.zotero.org/styles/presses-universitaires-de-paris-nanterre</id>
+    <link href="http://www.zotero.org/styles/presses-universitaires-de-paris-nanterre" rel="self"/>
+    <link href="http://scd.u-paris10.fr/services/boite-a-outils-pour-la-recherche-/ressources-pratiques-591002.kjsp?RH=1390906967545" rel="documentation"/>
+    <link href="https://presses.u-paris10.fr/wp-content/uploads/2017/02/ConsignesAuteurs.pdf" rel="documentation"/>
+    <author>
+      <name>Justine Ancelin</name>
+      <email>jancelin@u-paris10.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="social_science"/>
+    <category field="humanities"/>
+    <category field="generic-base"/>
+    <updated>2017-04-24T00:39:49+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="ordinal-01">re</term>
+      <term name="ordinal-02">e</term>
+      <term name="ordinal-03">e</term>
+      <term name="ordinal-04">e</term>
+      <term name="cited">op.&#160;cit.</term>
+      <term name="page" form="short">
+        <single>p.</single>
+        <multiple>p.</multiple>
+      </term>
+      <term name="editor" form="short">
+        <single>éd.</single>
+        <multiple>éd.</multiple>
+      </term>
+      <term name="container-author" form="short">
+        <single>dir.</single>
+        <multiple>dir.</multiple>
+      </term>
+      <term name="in">
+        <single>dans</single>
+        <multiple>dans</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="long" sort-separator=" " delimiter=", " and="text" delimiter-precedes-last="never" delimiter-precedes-et-al="never" et-al-min="4" et-al-use-first="4" name-as-sort-order="all" font-style="normal">
```

Comment:
@ajustine, `et-al-min` must always have a higher value than `et-al-use-first`. (`et-al-min` specifies the minimum number of names in a name list that is required to activate et-al abbreviation, and `et-al-use-first` is the number of names shown in an abbreviated name list before "et al.")